pkg/tsdb/elasticsearch/response_parser_test.go

```diff
@@ -1215,3 +1216,5 @@ func newResponseParserForTest(tsdbQueries map[string]string, responseBody string
 
 	return newResponseParser(response.Responses, queries, nil), nil
 }
+
+var update = flag.Bool("update", true, "update golden files")
```

Comment:
Nit: We usually place vars before their usage (before the test or at the top of the file for example)